src/agent/agentIntegration.ts

```diff
@@ -23,8 +23,8 @@ export async function getCommands(): Promise<(WizardCommandConfig | SimpleComman
         {
             type: "simple",
             name: createFunctionProjectCommandName,
-            commandId: "azureFunctions.createNewProject",
-            displayName: "Create Function Project",
+            commandId: "azureFunctions.createFunction",
+            displayName: "Create Function Locally",
```

Comment:
Do you remember if there was a reason we couldn't do something like `Create Function (Local)`?  I vaguely recall a reason but can't really remember anymore

Reply:
I don't know if there was a reason or not, but personally if I see (Local), it makes me think that there is another version of Create Function. Probably Create Function (Remote) or something like that.